src/control.js

```diff
@@ -0,0 +1,7 @@
+window.addEventListener('load', function() {
+    const btnStart = document.querySelector("#start");
+
+    btnStart.addEventListener("click", () => {
+        document.querySelector(".initial-container").classList.add("hide");
+    });
+});
```

Comment:
Can you perhaps add a newline at the end ?

Reply:
At which end?
The btnStart scope or the latest scope?

Reply:
in file `control.js` there is no blank line at the end.
*btw I'm an anime fan too, have seen detective conan* :smile:

Reply:
Done! :smile:

<i>Detective Conan :+1: </i>

Reply:
Perhaps could you send another PR and remove points from Todo in README.md which are covered in this PR ? :smile: 
Do you wish to become a collaborator ?

Reply:
I'll do the PR again.

Would really love to become a collaborator! :smile:

Reply:
Invite sent, check mail please :smile_cat:

Reply:
Accepted :smile: